ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -12,6 +12,8 @@ class PhaserProjectile extends Phaser.GameObjects.Container {
 		const key = `projectile/${projectile._stats.type}`;
 
 		const sprite = this.sprite = scene.add.sprite(0, 0, key);
+		sprite.displayWidth = projectile._stats.currentBody.width;
+		sprite.displayHeight = projectile._stats.currentBody.height;
```

Comment:
these look like great alternative to reading pixi object properties. 
are `currentBody` and its `width` and `height` always defined when creating a projectile? 

example: https://github.com/moddio/taro/blob/57ddd8db7c5e352ebb3ccce94492002e3a7c5a34/src/pixi/IgePixiAnimation.js#L59-L60

Reply:
Can you reference a time where `.currentBody` doesn't exist? These are the types of issues I was looking out for when we were doing Crash.

Reply:
not specifically, but existence of conditions in above code does indicate that there are cases when it doesn't exist.

Reply:
after doing some work with unit scaling I think the correct data is stored in `_bounds2d` as it gets updated when changing entity's width and height consistently